### ISSUE TITLE
Feature Request: Add possibility to pass ParquetOptions to ParquetReader in ParquetConvert.Deserialize

### ISSUE THREAD

**Issue by thomas-bios** [aloneguid/parquet-dotnet#174]
`ParquetConvert.Deserialize` create its own `ParquetReader` instance. But for some specific parquet files, we need to pass a specific option for the reading to succeed (`TreatByteArrayAsString = true` for example).

It could be interesting to either let the user give its own `ParquerReader` or to allow passing a `ParquetOptions`.

**Comment by cajuncoding** [comment]
Agree, the (_awesomely convenient_) `ParquetConvert` class could use some extra features on the APIs... this is def. a good idea as a optional feature.

I kinda like the idea of being able to provide your own Reader as a very flexible option, otherwise having a way to pass or configure the default options is another way to possibly go.

If I get some time in the coming weeks I may come back around to offering a PR with some improvements to these APS....

**Comment by aloneguid** [comment]
that would be great @cajuncoding

**Comment by cajuncoding** [comment]
Great! 👏
